News/DataAccess/Repo/TagRepo.cs

```diff
@@ -6,12 +6,35 @@ namespace News.DataAccess.Repo;
 
 public class TagRepo : ITagRepo
 {
-    private readonly DbSet<TagEntity> _dbSet;
     private readonly DbContext _context;
 
     public TagRepo(DbContext context)
     {
-        _dbSet = context.Set<TagEntity>();
         _context = context;
     }
-}
\ No newline at end of file
+
+    public TagEntity GetById(Guid id)
+    {
+        return _context.Set<TagEntity>().Find(id);        
+    }
+
+    public void Add(TagEntity entity)
+    {
+        _context.Add(entity);
+    }
+
+    public void Update(TagEntity entity)
+    {
+        _context.Update(entity);
+    }
+
+    public void Remove(TagEntity entity)
+    {
+        _context.Remove(entity);
+    }
+
+    public void SaveChanges()
```

Comment:
Do we need to create a method SaveChanges and use it in Services? We can do saving logic inside repos.